apply-pr.js

```diff
@@ -142,15 +142,14 @@ function resolveNames() {
     github.user.getFrom({ user: name }, resolve);
   });
 }
-
+console.log(OUTPUT['PR-URL'] + '.patch');
```

Comment:
Also you might want to assign `OUTPUT['PR-URL'] + '.patch'` to a variable to avoid hardcoding it in two places (the debug output and the actual location where it's used).